### PR TITLE
Updated README.md with my thought about OpenSource

### PR DESCRIPTION
**Pull request by bbsusheelkumar**
Thought by B B Susheel Kumar (http://bbsusheelkumar.github.io)
My first open source contribution ;) !!

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -454,3 +454,7 @@ Open source is important to me because it is where I learn a lot of stuff. I lea
 ### [Mareeswari Lakshmi](https://lakme22.github.io)
 Open source is a place where we can learn about team work. Its all about learning together in a community. Finally, I have learnt how to contribute to open source, a place from where I learnt so many things. It encourages me to share my knowledge and also to learn from others.
 
+### [B B Susheel Kumar](http://bbsusheelkumar.github.io)
+
+Open Source provides the possibility to build software as required by the community for the purpose of community and most importantly by the community. It is similar to the democracy in software development. Open Source provides the possibility of building a perfect and error or bug minimal software by allowing the discussion of the pros and cons based on the communities voice. It allows every individual to share their knowledge and also provide a platform to learn from many others. It goes by the famous quote of Helen Killer, "Alone we can do so little; together we can do so much". The greatest thing about Open Source is everyone contributes regardless of the credit just to make something good.
```
Comment (justinclift) [inline README.md]: "Helen Killer" -> "Helen Keller" :wink:

Comment (bbsusheelkumar) [reply README.md]: Done & Thank You :)

File: README.md [hunk 2]
```diff
@@ -454,3 +454,7 @@ Open source is important to me because it is where I learn a lot of stuff. I lea
...
 
+### [B B Susheel Kumar](http://bbsusheelkumar.github.io)
+
+Open Source provides the possibility to build software as required by the community for the purpose of community and most importantly by the community. It is similar to the democracy in software development. Open Source provides the possibility of building a perfect and error or bug minimal software by allowing the discussion of the pros and cons based on the communities voice. It allows every individual to share their knowledge and also provide a platform to learn from many others. It goes by the famous quote of Helen Keller, "Alone we can do so little; together we can do so much". The greatest thing about Open Source is everyone contributes regardless of the credit just to make something good.
+In a lighter side, the joy of work and happiness I get when ever my contributions to Open Source discussions, bugs and feature development gets accepted is not comparable to anything in the world. In the same way the knowledge and information which I get when when it gets rejected for certain reason is also enormous. It's like everything I do, whether correct or wrong, it makes me more better.
```
Comment (justinclift) [inline README.md]: Been thinking this one over too.  "make me more better." should really be "make me better.", as the "more" word there is redundant.

Yeah, I'm being kind of picky, but pretty much any native English speaker would pick up on that as being "bad wording".  Are you ok to update it?

Comment (bbsusheelkumar) [reply README.md]: Nooo, I would go with your suggestion.

File: README.md [hunk 3]
```diff
@@ -454,3 +454,7 @@ Open source is important to me because it is where I learn a lot of stuff. I lea
 ### [Mareeswari Lakshmi](https://lakme22.github.io)
 Open source is a place where we can learn about team work. Its all about learning together in a community. Finally, I have learnt how to contribute to open source, a place from where I learnt so many things. It encourages me to share my knowledge and also to learn from others.
 
+### [B B Susheel Kumar](http://bbsusheelkumar.github.io)
+
+Open Source provides the possibility to build software as required by the community for the purpose of community and most importantly by the community. It is similar to the democracy in software development. Open Source provides the possibility of building a perfect and error or bug minimal software by allowing the discussion of the pros and cons based on the communities voice. It allows every individual to share their knowledge and also provide a platform to learn from many others. It goes by the famous quote of Helen Keller, "Alone we can do so little; together we can do so much". The greatest thing about Open Source is everyone contributes regardless of the credit just to make something good.
```
Comment (nickdesaulniers) [inline README.md]: consider replacing `a perfect and error or bug minimal` with `optimal`.

File: README.md [hunk 4]
```diff
@@ -454,3 +454,7 @@ Open source is important to me because it is where I learn a lot of stuff. I lea
 ### [Mareeswari Lakshmi](https://lakme22.github.io)
 Open source is a place where we can learn about team work. Its all about learning together in a community. Finally, I have learnt how to contribute to open source, a place from where I learnt so many things. It encourages me to share my knowledge and also to learn from others.
 
+### [B B Susheel Kumar](http://bbsusheelkumar.github.io)
+
+Open Source provides the possibility to build software as required by the community for the purpose of community and most importantly by the community. It is similar to the democracy in software development. Open Source provides the possibility of building a optimal software by allowing the discussion of pros and cons based on the communities voice. It allows every individual to share their knowledge and also provide a platform to learn from many others. It goes by the famous quote of Helen Keller, "Alone we can do so little; together we can do so much". The greatest thing about Open Source is everyone contributes regardless of the credit just to make something good.
```
Comment (nickdesaulniers) [inline README.md]: s/a optimal/an optimal/`

**Comment by bbsusheelkumar** [comment]
Thank You Nick!! 🙂

**Pull request merged**